tsconfig.base.server.json

```diff
@@ -2,7 +2,7 @@
 	"extends": "./tsconfig.base.json",
 	"compilerOptions": {
 		"target": "es2020",
-		"lib": ["es2020"],
+		"lib": ["es2020", "dom", "dom.iterable"],
```

Comment:
IMO the "server" base tsconfig should not include `dom` and `dom.iterable` by default.. if it is required by package, it should be included in that package only.

see https://github.com/RocketChat/Rocket.Chat/blob/222fcc4f859970b3a76e9fedc0dd1817ba2b75a5/packages/cas-validate/tsconfig.json#L4

Reply:
So, found _why_ but not the real _why_. Seems like, for some reason, on this PR everything touching `core-typings` (which requires dom & dom.iterable) needs to have those options on build too. Still trying to find why it happens, or why it only happens on this CI tho

Reply:
I'll give it a try without it ;)